packages/next/test/unit/fixtures/02-edge-function-basepath/yarn.lock

```diff
@@ -0,0 +1,191 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+"@next/env@13.0.1":
```

Comment:
We might want to remove this lockfile too so we always test latest Next.js

Reply:
This contradicts the request to add a lockfile to the normalize paths PR: https://github.com/vercel/vercel/pull/8600. I think we should either keep the `yarn.lock` for the same reason it was added to the normalize paths PR or remove the lock file for both tests.

Reply:
Hmm good point, the lock file is going to be faster